libs/labelbox/src/labelbox/client.py

```diff
@@ -1204,9 +1221,65 @@ def create_ontology(self, name, normalized, media_type=None) -> Ontology:
                 'mediaType': media_type
             }
         }
+        if editor_task_type:
+            params['data']['editorTaskType'] = editor_task_type
+
         res = self.execute(query_str, params)
         return Entity.Ontology(self, res['upsertOntology'])
 
+    def create_model_chat_evaluation_ontology(self, name, normalized):
+        """
+        Creates a model chat evalutation ontology from normalized data
+            >>> normalized = {"tools" : [{'tool': 'message-single-selection', 'name': 'model output single selection', 'color': '#ff0000',},
+                                         {'tool': 'message-multi-selection', 'name': 'model output multi selection', 'color': '#00ff00',},
+                                         {'tool': 'message-ranking', 'name': 'model output multi ranking', 'color': '#0000ff',}]
+                             }
+            >>> ontology = client.create_ontology("ontology-name", normalized)
+
+        Or use the ontology builder
+            >>> ontology_builder = OntologyBuilder(tools=[
+                Tool(tool=Tool.Type.MESSAGE_SINGLE_SELECTION,
+                    name="model output single selection"),
+                Tool(tool=Tool.Type.MESSAGE_MULTI_SELECTION,
+                    name="model output multi selection"),
+                Tool(tool=Tool.Type.MESSAGE_RANKING,
+                    name="model output multi ranking"),
+            ],)
+
+            >>> ontology = client.create_model_chat_evaluation_ontology("Multi-chat ontology", ontology_builder.asdict())
+
+        Args:
+            name (str): Name of the ontology
+            normalized (dict): A normalized ontology payload. See above for details.
+        Returns:
+            The created Ontology
+        """
+
+        return self.create_ontology(
+            name=name,
+            normalized=normalized,
+            media_type=MediaType.Conversational,
+            editor_task_type=EditorTaskType.ModelChatEvaluation)
+
+    def create_model_chat_evaluation_ontology_from_feature_schemas(
+            self, name, feature_schema_ids):
```

Comment:
Can this function have its inputs + return types typed? I know we have a bunch of untyped methods but I think we should try to keep all new functions typed (admittedly we don't have a style guide to refer to or anything for this, I'd just really prefer it).

Reply:
yes